src/components/shared.module.css

```
@@ -5,6 +5,8 @@
   gap: var(--space-1);
 }
 
+.active {}
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)